### PR TITLE
Refactor docker compose

### PR DESCRIPTION
**Pull request by jhackettpps**

### CODE REVIEW

File: docker-compose.override.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+version: "3"
+
+services:
+    hub-node:
+        volumes:
+            - ./moj-node/server:/home/node/app/server
```
Comment (Tiki-God) [inline docker-compose.override.yml]: Do these volumes need mounting?  I can't find any volumes mounted in either staging or berwyn at the moment.

File: docker-compose.yml [hunk 2]
```diff
@@ -27,11 +25,7 @@ services:
                         APP_NAME: HMP Berwyn
```
Comment (Tiki-God) [inline docker-compose.yml]: Is it worth changing the default app name to something like "HMP Default".  That way we can easily pick up overrides that haven't been set properly?

File: docker-compose.override.yml (hunk 1 above)
Comment (Tiki-God) [reply docker-compose.override.yml]: As discussed these mounts are for local testing before staging.

File: docker-compose.yml (hunk 2 above)
Comment (jhackettpps) [reply docker-compose.yml]: Yeah that could work well actually, will make that change

**Pull request merged**